### PR TITLE
CAM-12084: Detail Export documented.

### PR DESCRIPTION
**Pull request by Victor-5**

### CODE REVIEW

**Comment by Victor-5** [comment]
@gsanchezu @jinzishuai Please review

File: Cloud Application Manager/Core Concepts/providers.md [hunk 1]
```diff
@@ -9,14 +9,15 @@
...
 
-* [Overview](#overview)
-* [Audience](#audience)
-* [Prerequisites](#prerequisites)
-* [Providers](#providers)
-* [Providers page](#providers-page)
-* [Synchronization process](#synchronization-process)
-* [Supported provider types](#supported-provider-types)
-* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+- [Overview](#overview)
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]: Please, leave `*` as the bullet points symbol

File: Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md [hunk 2]
```diff
@@ -8,17 +8,38 @@
...
-* [Prerequisites](#prerequisites)
-* [Instances page](#instances-page)
-* [Deploying a New Instance](#deploying-a-new-instance)
-* [Scheduling Instances](#scheduling-instances)
-* [Protecting Instance Shutdown or Termination](#protecting-instance-shutdown-or-termination)
-* [Deploying Instances without Lifecycle Management](#deploying-instances-without-lifecycle-management)
-* [Handling Instance Lifecycle States](#handling-instance-lifecycle-states)
-* [Managing multiple instances with bulk actions](#managing-multiple-instances-with-bulk-actions)
-* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+- [Overview](#overview)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md]: Please, leave * as the bullet points symbol, and only show 3 level titles in the ToC

File: Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md [hunk 3]
```diff
@@ -308,6 +329,24 @@ Many of the instance actions can be applied in bulk to several instances at once
...
+
+Clicking this button toggles a drop-down menu with two options:
+- Export to CSV
+- Export to PDF
+
+![Detail Export menu](../../images/cloud-application-manager/deploying-anywhere/detail-export-instances-01.png)
+
+When one of these options is selected, a file is downloaded to your system and saved based on your browser's settings (by default, in the Downloads folder). The file will be named _instances.csv_ or _instances.pdf_; or, if a file by that name already exists, the name will be modified according to your operating system rules (e.g., _instances (1).csv_).
+
+Please note that **all** instances in the current scope are exported, not necessarily just the ones you see in the list. The current scope is either the currently selected workspace, or the cost center, or the organization. For a workspace, all the instances that are shared with the workspace are also included.
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md]:
```suggestion
Please note that **all** instances in the current scope are exported, not necessarily just the ones you see in the list (no applied filters are considered). The current scope is either the currently selected workspace, or the cost center, or the organization selected in the context switcher. For a workspace, all the instances that are shared with the workspace are also included.
```

File: Cloud Application Manager/Core Concepts/providers.md [hunk 4]
```diff
@@ -114,6 +115,24 @@ Cloud Application Manager integrates with many popular clouds. Here is a full li
...
+
+Clicking this button toggles a drop-down menu with two options:
+- Export to CSV
+- Export to PDF
+
+![Detail Export menu](../../images/cloud-application-manager/deploying-anywhere/detail-export-providers-01.png)
+
+When one of these options is selected, a file is downloaded to your system and saved based on your browser's settings (by default, in the Downloads folder). The file will be named _providers.csv_ or _providers.pdf_; or, if a file by that name already exists, the name will be modified according to your operating system rules (e.g., _providers (1).csv_).
+
+Please note that **all** providers in the current scope are exported, not necessarily just the ones you see in the list. The current scope is either the currently selected workspace, or the cost center, or the organization. For a workspace, all the providers that are shared with the workspace are also included.
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]:
```suggestion
Please note that **all** providers in the current scope are exported, not necessarily just the ones you see in the list (no applied filters are considered). The current scope is either the currently selected workspace, or the cost center, or the organization selected in the context switcher. For a workspace, all the providers that are shared with the workspace are also included.
```

**Pull request merged**